.github/workflows/e2e.yml

```diff
@@ -0,0 +1,84 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+# graphic logo are (registered/a) trademark(s) of Plan International.
+name: End-to-end tests
+
+on:
+  push:
+    branches:
+      - 'develop'
+
+jobs:
+  E2E:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    env:
+      CYPRESS_LOGIN_URL: http://localhost:3020/
+      CYPRESS_CLIENT_URL: http://localhost:3000/
+      CYPRESS_AUTH_URL: http://localhost:4040/
+
+    strategy:
+      matrix:
+        node-version: [14.17.0]
+
+    steps:
+      - name: Checking out git repo
+        uses: actions/checkout@v2
+
+      - name: Generate Secrets
+        run: |
+          openssl genrsa -out .secrets/private-key.pem 2048
+          ls -la .secrets
+          openssl rsa -pubout -in .secrets/private-key.pem -out .secrets/public-key.pem
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+
+      - name: Install docker
+        uses: docker-practice/actions-setup-docker@master
+
+      - name: Install dependencies
+        run: |
+          yarn
+          yarn global add wait-on
+
+      - name: Initialize docker & folder structures
+        run: |
+          docker swarm init
+          sudo mkdir -p data/elasticsearch
+          sudo chown -R 1000:1000 data/elasticsearch
+
+      - name: Download & run docker images
+        run: |
+          screen -d -m yarn compose:deps
+          wait-on -l tcp:3447 http://localhost:9200 tcp:5001 tcp:9200 tcp:27017 tcp:6379 tcp:8086
+
+      - name: Run Microservices
+        run: |
+          screen -dm bash -c 'LANGUAGES=en yarn start'
+          wait-on -l tcp:4040 http://localhost:3000 http://localhost:3020 tcp:3030 tcp:2020 tcp:7070 tcp:5050 tcp:9090 tcp:1050
+
+      - name: Running the Country specific resource package
+        run: |
+          git clone -b ocrvs-2339 --depth=1 https://github.com/opencrvs/opencrvs-zambia.git
```

Comment:
Do we need to checkout the `ocrvs-2339` branch here?

Reply:
I wonder if there would be any chance to get this running with develop version of Farajaland 🤔

Reply:
Trying it out here: https://github.com/opencrvs/opencrvs-core/actions/runs/1744065769

Reply:
It would be good if e2e tests could be triggered manually for any branch with config also for resource package repo address and branch